util/util.go

```diff
@@ -20,12 +20,12 @@ func JsonDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	var oldJSON, newJSON interface{}
 	err := json.Unmarshal([]byte(old), &oldJSON)
 	if err != nil {
-		log.Printf("[ERROR] Version of %q in state is not valid JSON: %s", k, err)
+		log.Printf("[WARN] Version of %q in state is not valid JSON: %s", k, err)
```

Comment:
Why do we no longer want to log errors here?

Reply:
Covered in the changelog and the PR's description.
```
update util.JsonDiffSuppress to log at the warn level rather than
error, since the JSON parse errors are treated as non fatal
```

Reply:
Cool, sorry I missed that